src/sentry/tasks/post_process.py

```diff
@@ -994,6 +994,10 @@ def process_rules(job: PostProcessJob) -> None:
     from sentry.rules.processing.delayed_processing import process_delayed_alert_conditions
 
     if not redis_buffer_registry.has(BufferHookEvent.FLUSH):
+        logger.info(
+            "post_process.process_rules.adding_bufferhookevent",
+            extra={"key_name": BufferHookEvent.FLUSH.value},
```

Comment:
unclear if it's helpful... but maybe we should also serialize the function here too? i'm mostly thinking that i'd be nice to see as much information as possible.